### DISCUSSION TITLE
Has anyone been able to achieve this level of nested sidebars with Shadcn?

### DISCUSSION THREAD

**Discussion by VenoM9078** [shadcn-ui/ui#5622] [General]
![image](https://github.com/user-attachments/assets/5f5bb030-138c-49b1-9657-2378b34c1fa7)
![image](https://github.com/user-attachments/assets/fc45bd84-b01d-4af3-a7e6-30cfa5106b83)

So essentially, the left most sidebar remains an icon static sidebar, collapse/expand has no effect on it. The other two sidebars, expand and collapse to icons.

If anyone has achieved this with the new sidebar component and is willing to share some idea on how to do it - that'd be great, thanks!

**Comment by johnweland** [comment]
I haven't tried it but I noticed when I built a layout with 2 sidebars one on the left one on the right and use the sidebar trigger, since both are collapsible, both fire on the same trigger. I imagine you could run three sidebars all on the same side. 1st not collapsible but icons only the other two set to collapsible with icon as its variant.

**Comment by JolomiTee** [comment]
Hi @VenoM9078  you can check this issue thread out: https://github.com/shadcn-ui/ui/issues/5651

**Comment by Dio-brandio** [comment]
Not 3 but 2 sidebar 
[here](https://dual-sidebar-shadcn.vercel.app/dashboard)

**Comment by talhanr** [comment]
yes i did it.

**Reply by spacerrsdreams** [reply]
Wanna share?
